### PR TITLE
chore: remove findbugs workaround; take advantage of checkstyle defaults

### PR DESCRIPTION
**Pull request by keturn**
also upgrade gradle while we're here because the config is tiny and thus really hard to break.

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-rc-1-all.zip
```
Comment (skaldarnar) [inline gradle/wrapper/gradle-wrapper.properties]: Is there a reason why we depend on a release candidate of Gradle here?

Comment (keturn) [reply gradle/wrapper/gradle-wrapper.properties]: I was feeling optimistic about being able to target 7.5 after the RC came out.

Now that it's been 3+ weeks with no release or RC2, that may have been overly optimistic…

Comment (jdrueckert) [reply gradle/wrapper/gradle-wrapper.properties]: so shall we go with 7.4 and get this merged then?

Comment (skaldarnar) [reply gradle/wrapper/gradle-wrapper.properties]: I'm fine with merging this if we go for the latest **released version** right now :upside_down_face:

**Pull request merged**